src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -59,6 +60,8 @@ public void init(FeedbackSessionAttributes feedbackSession, List<FeedbackQuestio
 
         buildPreviewForm(feedbackSession, studentList, instructorList);
 
+        this.shouldLoadInEditMode = shouldLoadInEditMode;
+
```

Comment:
Sorry I was unclear; I meant the blank line as it's unrelated. The assignment looks like it's necessary?